src/main/java/org/fundacionjala/coding/ariel/StrongNumber.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.ariel;
+
+/**
+ * A number is called strong number if the sum of the factorial of its digits is equal to number itself.
+ * <p>
+ * For example: 145 since: 1! + 4! + 5! = 1 + 24 + 120 = 145.
+ * strong_num (1)    return "STRONG!!!!"
+ * strong_num (2)    return "STRONG!!!!"
+ * strong_num (123)  return "Not Strong !!"
+ * strong_num (150)  return "Not Strong !!"
+ */
+public class StrongNumber {
+
+    /**
+     * The constructor.
+     */
+    public StrongNumber() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)